react/components/molecules/tabs-text/tabs-text.js

```diff
@@ -21,6 +21,7 @@ export class TabsText extends PureComponent {
             variant: PropTypes.string,
             parentWidth: PropTypes.number,
             onTabChange: PropTypes.func.isRequired,

```

Comment:
Why can't you use this existing callback?

Reply:
Because for me it didn't make much sense call `onTabChange` when technically the tab is not being changed, that's why a put wrapped the methods in a condition

Reply:
Fair enough 👍